package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "exp-config",
   "description": "Simple configuration management",
-  "version": "4.2.2",
+  "version": "4.2.3",
```

Comment:
For the future, this should really be a major bump as dropping old node versions introduces a breaking change